build.gradle

```diff
@@ -0,0 +1,21 @@
+apply plugin: 'java'
+apply plugin: 'war'
+
+repositories {
+    jcenter()
+    mavenCentral()
+    mavenLocal()
+}
+
+dependencies {
+    compile fileTree(dir: 'lib', include: '*.jar')
+    compile 'javax.servlet:javax.servlet-api:3.0.1'
+    compile 'postgresql:postgresql:9.1-901.jdbc4'
+
+    testCompile 'simple-jndi:simple-jndi:0.11.4.1'
+    testCompile 'junit:junit:4.12'
+}
+
+sourceSets.main.java.srcDirs = ["src"]
+// the tests fail because they have environemnt specific parameters
+// sourceSets.test.java.srcDirs = ["test"]
```

Comment:
For example, the one you proposed to Alberto in the other issue:

```
SELECT c, distance(c, point('', 187.48, 2.05))
FROM (SELECT TOP 10 centroid(s_region) AS c FROM ivoa.ObsCore) AS trick
```

in this case colType is "UNKNOWN".